indico/modules/events/papers/controllers/management.py

```diff
@@ -111,3 +114,28 @@ def _process(self):
             return jsonify_data()
         return WPManagePapers.render_template('management/competences.html', self._conf, event=self.event_new,
                                               form=form)
+
+
+class RHManageReviewingSettings(RHManagePapersBase):
+    def _process(self):
+        defaults = FormDefaults(content_review_questions=self.event_new.cfp.content_review_questions,
+                                layout_review_questions=self.event_new.cfp.layout_review_questions,
+                                **paper_reviewing_settings.get_all(self.event_new))
+        form = PaperReviewingSettingsForm(event=self.event_new, obj=defaults)
+        if form.validate_on_submit():
+            data = form.data
+            content_review_questions = data.pop('content_review_questions', [])
+            layout_review_questions = data.pop('layout_review_questions', [])
+            self.event_new.paper_review_questions = []
```

Comment:
That's not how this works ;)

By doing this, SQLAlchemy may delete the existing questions, and thus the associated answers (or try to delete them and fail if no cascading is set up). Either way, it's not what you want.

I see you already added a field similar to the one I added in the abstract dialog, but you need to use logic similar to what I did in `RHManageAbstractReviewing` too (do the assignment to deal with new questions and also - very important - don't commit in case the form wasn't submitted successfully to avoid committing editions/deletions if form validation failed somewhere else).